File: aula06/atividade05/Usuario.java
package atividade05;

import java.util.Scanner;

public class Usuario {
    String senha;
    String nome;
    boolean isLogado;

    public Usuario() {
        this.senha = "admin";
        this.nome = "admin";
        this.isLogado = false;
    }

    public void logar() {
        Scanner scanner = new Scanner(System.in);

        while (true) {
            System.out.print("\n Nome: ");
            String nome = scanner.nextLine();

            System.out.print("\n Senha: ");
            String senha = scanner.nextLine();
            
            if (nome.equals(this.nome) && senha.equals(this.senha)) {
                this.isLogado = true;
                System.out.println("\n Usuário " + this.nome + " logado...\n");
                break;
            }
            else {
                System.out.println("\n Usuário ou senha inválidos !!!");
            }
        }

        scanner.close();
    }

    public static void main(String[] args) {
        Usuario usuario = new Usuario();
        usuario.logar();
    }
}

File: aula06/atividade05/Usuario.py
class Usuario:
    def __init__(self, nome, senha) -> None:
        self.senha = senha
        self.nome = nome
        self.isLogado = False

    @classmethod
    def criar_usuario_admin(cls):
        return cls('admin', 'admin')

    def logar(self):
        while True:
            nome = input('\n Nome: ')
            senha = input(' Senha: ')

            if nome == self.nome and senha == self.senha:
                self.isLogado = True
                print(f'\n Usuário {self.nome} logado...\n')
                break
            else:
                print('\n Usuário ou senha inválidos !!!')

if __name__ == '__main__':
    usuario_admin = Usuario.criar_usuario_admin()
    usuario_admin.logar()
